### DISCUSSION TITLE
Landing without gears

### DISCUSSION THREAD

**Discussion by eeeewwqq** [JSBSim-Team/jsbsim#958] [Q&A]
I'm trying to implement landing without gears. And want to add specific sound to this action.
How to know is `<ground_reactions><contact type="STRUCTURE" name="NOSE_BOTTOM">` have contact with ground?

**Comment by seanmcleod** [comment]
Take a look at the `WOW` (Weight On Wheel) property which applies to `STRUCTURE` elements as well.

https://github.com/JSBSim-Team/jsbsim/blob/c7cd8ef03508b7a8d58a717aec4255410897a2e4/src/models/FGLGear.cpp#L772-L784

**Reply by eeeewwqq** [reply]
Thanks, it worked.

I was looking for WOW but in gear/unit[number]/wow. Because property gear/num-units return number of all contact types (bogey + structure). Is there way to get true number of gears?

**Comment by seanmcleod** [comment]
Hmm, does look like the naming of the function/property is a bit misleading.

What language are you using for your logic while querying these properties? Python, Nasal script?

You could potentially loop over all `gear/num-units` querying `gear/unit[i]/` etc. and catch exceptions etc. to figure out exactly how many landing gears/bogeys there are and which indices they populate etc.

**Reply by eeeewwqq** [reply]
I'm using unity c sharp.

Yeah, I implemented it like you wrote above. Just thought may be exists some property to get real number of gears.
